Example/Pageboy-Example/PageViewController.swift

```diff
@@ -104,14 +104,17 @@ extension PageViewController: PageboyViewControllerDataSource {
 
 // MARK: PageboyViewControllerDelegate
 extension PageViewController: PageboyViewControllerDelegate {
-    
     func pageboyViewController(_ pageboyViewController: PageboyViewController,
                                willScrollToPageAt index: PageboyViewController.PageIndex,
                                direction: PageboyViewController.NavigationDirection,
                                animated: Bool) {
 //        print("willScrollToPageAtIndex: \(index)")
     }
     
+    func pageboyViewController(_ pageboyViewController: PageboyViewController, abortScrollToPageAt index: PageboyViewController.PageIndex, returnToPageAt previousIndex: PageboyViewController.PageIndex) {
```

Comment:
I think a default implementation might be a good idea here - don't want to introduce a breaking API change and also it is more of a niche use than the other delegate functions.

Reply:
And done.